### ISSUE TITLE
4.3 - Main menu music has stopped working

### ISSUE THREAD

**Issue by Maaack** [Maaack/Godot-Game-Template#128]
> ... Since godot 4.3 is out, the music don't work anymore in the mainMenu from your addon.
> but you can fix it by setting discrete mode to "dominant" in animation_tree
-- Ryokugin

![image](https://github.com/user-attachments/assets/e144aeeb-0535-4a15-a191-2c42a148ba17)

**Comment by Maaack** [comment]
I did manage to reproduce this bug once and started trying to debug it. Since then, I've tried recreating it on two more fresh projects with no luck.

From what I could tell, the bug was occurring because Godot was not editing the file it claimed to be. I noticed when I opened and added a stream to the BackgroundMusicPlayer of `MainMenuWithAnimations.tscn` in my `scenes/` directory, the actual changes were getting applied to the original copy in the `addons/.../examples/` directory. I believe this is a result of UIDs getting duplicated, or not updating correctly during the copy process. Deleting the original `example/` files might be enough to fix the issue, or deleting the copied files and reattempting to copy should also work.
